src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -74,14 +77,19 @@ public string GetLastRequestHeader()
                 failedRequests.Append(",");
                 failedRequests.Append(ev.CorrelationId);
 
+                if (!firstFailure)
+                    platformFields.Append(",");
+
+                platformFields.Append(region);
+
                 firstFailure = false;
             }
 
             string data =
                 $"{TelemetryConstants.HttpTelemetrySchemaVersion2}|" +
                 $"{_successfulSilentCallCount}|" +
                 $"{failedRequests}|" +
-                $"{errors}|";
+                $"{errors}|{platformFields}";
```

Comment:
>|{platformFields} [](start = 26, length = 17)

nit: would have preferred this to be on a separate line

Reply:
Done

---
In reply to: [492936649](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2042#discussion_r492936649) [](ancestors = 492936649)